### DISCUSSION TITLE
Support of FB2 format

### DISCUSSION THREAD

**Discussion by gimrock** [sumatrapdfreader/sumatrapdf#2301] [General]
Sumatra needs better FB2 support.
Here are some problem I noticed:

Doesn’t open new chapter on the new page.
Doesn’t distinguish epigraphs.
Doesn’t see text of the picture.
Doesn’t distinguish quote and its author from common text.
Doesn’t recognize poems.
Doesn’t display notes and commentaries on the bottom.
Doesn’t recognize tables.
And it looks better when titles are placed in the center of page.

Here you go:
![sumatra](https://user-images.githubusercontent.com/54965622/151707476-cf6b01f3-956f-4a45-b449-a98adf52ecac.jpg)

And how the same page looks for example in AlReaderX app:
![alreaderx](https://user-images.githubusercontent.com/54965622/151707486-8d0b72b6-8615-4925-9c49-8257d029d007.jpg)

One more thing: reader doesn’t recognize SUB and SUP text:
![subtext](https://user-images.githubusercontent.com/54965622/151707514-d7f0f8f0-6f6d-4cd1-8073-539f5eb597a6.jpeg)

The example of FB2 file:
[file_example](https://github.com/sumatrapdfreader/sumatrapdf/files/7966326/On.Guard.defending.your.faith.with.reason.and.precision.by.William.Lane.Craig.z-lib.org.epub.fb2.zip)

**Comment by kjk** [comment]
Almost all those problems are fixed if:
* you use latest [pre-release version](https://www.sumatrapdfreader.org/prerelease)
* open the actual .fb2 file, not .zip file with .fb2 inside

Pre-release has a new engine for .fb2 that supports more advanced formatting but it's not activated for .fb2 files inside .zip

I've opened https://github.com/sumatrapdfreader/sumatrapdf/issues/2305 to improve that

**Reply by gimrock** [reply]
I zipped FB2 file to add it to the post. I didn't open it in an archive on my PC.

I tryed the pre-release version:

Items 2, 3 and 6 are not fixed yet.
Item 4 - no selection for an `<text-author>` tag.
Item 7 - no borders for the table

And I can't find ebookUI in pre-release Advanced Options.

In general, Sumatra is very good. Thank you for amazing work.
